pandapipes/plotting/simple_plot.py

```diff
@@ -284,24 +284,24 @@ def create_simple_collections(net, respect_valves=False, respect_in_service=True
             circ_pump_colls = create_pump_collection(
                 net, pumps=net.circ_pump_mass[net.circ_pump_mass.in_service].index,
                 table_name='circ_pump_mass', size=pump_size, linewidths=pipe_width,
-                color=pump_color)
+                color=pump_color, fj_col="return_junction", tj_col="flow_junction")
```

Comment:
Would it make sense instead of passing the junction names, to pass the components itself, as the components have the corresponding class method. So in case we change the name again, there is no need of adaptiong the code here.